Description: Remove Google AdSense from the docs
 Debian-specific patch because of Lintian warnings about
 the use of Google AdSense in some HTML files
Author: Joachim Wiedorn <joodebian at joonet.de>
Forwarded: no
Last-Update: 2016-03-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff -urN s09/doc/download.html s10/doc/download.html
--- s09/doc/download.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/download.html	2014-01-04 20:57:51.568546972 +0100
@@ -121,18 +121,6 @@
 </ul>
 
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
-
 <!--- COPYRIGHT -->
 <p>
 <table width=100% cellpadding=0 cellspacing=0><tr><td width=100% valign=top id=HEADLINE align=right>
diff -urN s09/doc/faq.html s10/doc/faq.html
--- s09/doc/faq.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/faq.html	2014-01-04 20:58:28.514692930 +0100
@@ -2064,18 +2064,6 @@
 <img src='art/line.gif' width=100% height=1>
 </td></tr></table>
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
-
 
 <!--- COPYRIGHT -->
 <p>
diff -urN s09/doc/home.html s10/doc/home.html
--- s09/doc/home.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/home.html	2014-01-04 20:58:42.966149231 +0100
@@ -122,18 +122,6 @@
 </ul>
 
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
-
 <!--- COPYRIGHT -->
 <p>
 <table width=100% cellpadding=0 cellspacing=0><tr><td width=100% valign=top id=HEADLINE align=right>
diff -urN s09/doc/install.html s10/doc/install.html
--- s09/doc/install.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/install.html	2014-01-04 20:59:00.649581683 +0100
@@ -591,17 +591,6 @@
 </pre>
 </ul>
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
 
 <!--- COPYRIGHT -->
 <p>
diff -urN s09/doc/news1.html s10/doc/news1.html
--- s09/doc/news1.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/news1.html	2014-01-04 20:59:33.514339255 +0100
@@ -2060,17 +2060,6 @@
   <li>Listing files, drive letters, and network neighborhood (Windows) have been added, and there have been many bug fixes.</li>
 </ul>
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
 
 <!--- COPYRIGHT -->
 <p>
diff -urN s09/doc/news2.html s10/doc/news2.html
--- s09/doc/news2.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/news2.html	2014-01-04 20:59:48.647529654 +0100
@@ -868,17 +868,6 @@
 <!--- TOPIC TITLE -->
 <a href="news1.html">Older news...</a>
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
 
 <!--- COPYRIGHT -->
 <p>
diff -urN s09/doc/news3.html s10/doc/news3.html
--- s09/doc/news3.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/news3.html	2014-01-04 21:00:00.713181969 +0100
@@ -1254,17 +1254,6 @@
 <!--- TOPIC TITLE -->
 <a href="news2.html">Older news...</a>
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
 
 <!--- COPYRIGHT -->
 <p>
diff -urN s09/doc/news.html s10/doc/news.html
--- s09/doc/news.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/news.html	2014-01-04 20:59:19.704402130 +0100
@@ -196,17 +196,6 @@
 <!--- TOPIC TITLE -->
 <a href="news3.html">Older news...</a>
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
 
 <!--- COPYRIGHT -->
 <p>
diff -urN s09/doc/projects.html s10/doc/projects.html
--- s09/doc/projects.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/projects.html	2014-01-04 21:00:37.351223534 +0100
@@ -469,18 +469,8 @@
 If you want <em>your</em> FOX based application to be featured here, or need to update the
 above information, please mail <a href="mailto:jeroen@fox-toolkit.org">me</a>
 and I'll update this page!
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript"
-  src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
+
+
 <!--- COPYRIGHT -->
 <p>
 <table width=100% cellpadding=0 cellspacing=0><tr><td width=100% valign=top id=HEADLINE align=right>
diff -urN s09/doc/win32.html s10/doc/win32.html
--- s09/doc/win32.html	2013-04-07 20:51:42.000000000 +0200
+++ s10/doc/win32.html	2014-01-04 21:00:56.430111784 +0100
@@ -304,19 +304,6 @@
 
 </ul>
 
-<p>
-<center>
-<script type="text/javascript"><!--
-google_ad_client = "pub-3898868524945843";
-google_ad_width = 728;
-google_ad_height = 90;
-google_ad_format = "728x90_as";
-//--></script>
-<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
-</script>
-</center>
-
-
 
 <!--- COPYRIGHT -->
 <p>
